src/Evolution/Executables/ScalarWave/EvolveScalarWave.hpp

```diff
@@ -148,15 +150,16 @@ struct EvolutionMetavars {
                      Tags::Time, analytic_solution_fields>,
                  Events::Registrars::ObserveTimeStep<EvolutionMetavars>,
                  Events::Registrars::ChangeSlabSize<slab_choosers>>;
-  using triggers = Triggers::time_triggers;
 
   using observed_reduction_data_tags = observers::collect_reduction_data_tags<
       typename Event<events>::creatable_classes>;
 
   struct factory_creation
       : tt::ConformsTo<Options::protocols::FactoryCreation> {
     using factory_classes = tmpl::map<
-        tmpl::pair<StepController, StepControllers::standard_step_controllers>>;
+        tmpl::pair<StepController, StepControllers::standard_step_controllers>,
+      tmpl::pair<Trigger, tmpl::append<Triggers::logical_triggers,
```

Comment:
(optional) this formatting is off. perhaps run `git-clang-format upstream/develop` over this branch.